test/Resources/TestProjects/DotNetCustomHandler/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "profiles": {
+    "DotNetCustomHandler": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "https://localhost:56495;http://localhost:56496"
+    }
+  }
+}
```

Comment:
I think this was committed/added by mistake?

Reply:
Yeah, _something_ keeps auto generating this file on me.